docs/asciidoc/debugging.adoc

```diff
@@ -104,6 +104,10 @@ the `Flux`.
 [[debug-activate]]
 == Activating Debug Mode
 
+WARNING: this section describes the easiest but also the slowest way to enable
+the debugging capabitilies due to the fact that it captures the stacktrace on every operator.
+See <<reactor-tools-debug>> for more advanced and performant option.
```

Comment:
suggestion: also mention the `checkpoint alternative` section:
```
See <<checkpoint-alternative>> for a more fine grained way of debugging, and <<reactor-tools-debug>> for a more advanced and performant global option.
```

⚠️ **You will need to add the `[[checkpoint-alternative]]` anchor to line 261**